.vscode/launch.json

```diff
@@ -1,4 +1,5 @@
 {
+    "version": "0.2.0",
```

Comment:
I just added this line out of curiosity. My testing with vscode was crashing without this (I could not call the debugger). Is there a reason why did you remove this from the default?

Reply:
Oh interesting -- I didn't edit it at all, this is what my vscode setup generated. Maybe we shouldn't even have these files checked into the repo at all?

Reply:
Yep, they probably should not be there.